Test.groovy

```diff
@@ -17,9 +17,12 @@ pipeline {
       }
     }
     stage('Start GraphQL engine') {
+      environment {
+        API_PORT = 8081
```

Comment:
We don't care about what port we use, as long as it is not `8080`. It's pure configuration, should be easy to do even in a development environment, therefore I thought using an environment variable would do the trick.